doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst

```diff
@@ -18,12 +18,12 @@ Features
 - Interface sharpening
 
 
-------------------------
-Location of the example
-------------------------
-``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-constant-refinement.prm``
+--------------------------------------------
+Location of the files used in this example
+--------------------------------------------
+``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability_constant_sharpening.prm``
 
-``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-adaptive-refinement.prm``
+``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability_adaptive_sharpening.prm``
```

Comment:
I don't remember if we want - or _ in the file name, but either way, the name of the file in the .rst has to match with the file in the example folder. If it is _ (e.g., rayleigh-taylor-instability_constant_sharpening.prm), we have to rename the .prm :) 

We can also reproduce what is done in the order example and just refer to the example folder:
```suggestion
``examples/multiphysics/rayleigh-taylor-instability/``
```